kristofer.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("Hello, Kristofer")
```

Comment:
Could you please add a comment here!

Reply:
Sure hang on!

Reply:
Comment has been added, please let me know if I can help with anything else